Header header
uint16 led_number 
uint8 red
uint8 green
uint8 blue

